language-translator/src/main/java/com/ibm/watson/developer_cloud/language_translator/v2/LanguageTranslator.java

```diff
@@ -37,16 +37,22 @@
 import okhttp3.RequestBody;
 
 /**
+ *
+ *
+ * ---
+ *
+ * Language Translator v3 is [available](https://www.ibm.com/watson/developercloud/language-translator/api/v3/). See the
+ * [migration guide](https://console.bluemix.net/docs/services/language-translator/migrating.html).
+ *
+ * ---
+ *
  * IBM Watson&trade; Language Translator translates text from one language to another. The service offers multiple
  * domain-specific models that you can customize based on your unique terminology and language. Use Language Translator
  * to take news from across the globe and present it in your language, communicate with your customers in their own
  * language, and more.
  *
  * @version v2
  * @see <a href="http://www.ibm.com/watson/developercloud/language-translator.html">Language Translator</a>
- * @deprecated Language Translator v3 is now available. The v2 Language Translator API will no longer be available
```

Comment:
Are we replacing this deprecation message with the one above?

Reply:
I wasn't sure.. The one thing with the annotation is that it will actually show in IDEs as a warning to developers. So I think it's helpful to have.